spec/models/player_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Player, type: :model do
+  let(:game) { Game.new }
+  subject(:player) { Player.new game: game }
+  
+  describe "#plays_in?" do
+    it "is true if a player is part of the given game", :aggregate_failures do
+      expect(player.plays_in?(game)).to be_truthy
```

Comment:
Yep so I was saying in your other branch that I prefer to be explicit here with `eq(true)` / `eq(false)`.
We expect that it's either `true` or `false` and not `truthy` / `falsy`.
What do you think?